src/Uno.UI/ts/Windows/Media/SpeechRecognizer.ts

```diff
@@ -0,0 +1,44 @@
+﻿namespace Windows.Media {
+	export class SpeechRecognizer {
+
+		private static dispatchMessage:
+			(instanceId: string, serializedMessage: string, timestamp: number) => number;
+
+		private static instanceMap: { [managedId: string]: SpeechRecognizer } = {};
+
+		private managedId: string;
+		private recognition: SpeechRecognition;
+
+		private constructor(managedId: string) {
+			this.managedId = managedId;
+			this.recognition = new window.SpeechRecognition();
```

Comment:
Maybe you should test for the availability of this feature before using it.

Reply:
Added :-)